projects/plugins/jetpack/extensions/blocks/google-docs-embed/preview.js

```diff
@@ -0,0 +1,55 @@
+import { __ } from '@wordpress/i18n';
+
+const Preview = props => {
+	const {
+		allowFullScreen = true,
+		isPrivateURL = false,
+		interactive,
+		toggleInteractive,
+		title = __( 'Embed URL', 'jetpack' ),
+		url,
+	} = props;
+
+	return (
+		/* this extra wrapper div gets max-width set */
+		<figure>
+			{ isPrivateURL ? (
+				<div className="wp-block-jetpack-google-docs-embed__wrapper wp-block-jetpack-google-docs-embed__wrapper--error">
+					<p className="wp-block-jetpack-google-docs-embed__error-msg">
+						{ __( 'This Google Document is private.', 'jetpack' ) }
+						<br />
+						<br />
+						<a href={ url } target="_blank" rel="noreferrer">
```

Comment:
Should we use `ExternalLink` here? ([reference](https://developer.wordpress.org/block-editor/reference-guides/components/external-link/))

Reply:
Updated.